repos/spack_repo/builtin/packages/qt/package.py

```diff
@@ -195,6 +195,9 @@ class Qt(Package):
         when="@5.9.2: %gcc@14:",
     )
 
+    # Do not define `wtf_ceil()` in MathExtras.h on macOS
```

Comment:
sorry, but on second thought, can we also expand this docstring a bit? basically just describe
What is does (you did that)
Why it does that
Why we need that patched
i.e. Prevents references to removed api, required to prevent compilation errors for webkit on macos, etc